docs/daemon/snapshot.md

```diff
@@ -38,3 +38,19 @@ Snapshot current node's latest data
 ```bash
  iris snapshot --home=<path-to-your-home>
 ```
+
+## FAQ
+
+### What happens if everyone delete previous stored blockchain data and a new machine will try to sync up
```

Comment:
What if every node in the network runs from a snapshot and a new node tries to catch up from genesis